src/Model/Scope/Condition.php

```diff
@@ -87,12 +121,20 @@ public function __construct($key, $operator = null, $value = null)
 
         if (func_num_args() == 2) {
             $value = $operator;
-            $operator = '=';
+            $operator = self::OPERATOR_EQUALS;
         }
 
         $this->key = $key;
-        $this->operator = $operator;
         $this->value = $value;
+
+        if (isset($operator)) {
```

Comment:
`$operator !== null` - check also other code, I see the same a few lines below

`isset()` should do the same, expect `isset($iInventedNonDeclaredVariable)` typos will be supressed - very, very dangerous!

Reply:
I see. Changed ...